platform-common-date/src/main/java/com/softicar/platform/common/date/Day.java

```diff
@@ -197,15 +197,16 @@ public static Day get1970() {
 	// -------------------------------- FACTORY FUNCTIONS -------------------------------- //
 
 	/**
-	 * Returns the specified day of the given year and month.
+	 * Returns the {@link Day} for the given triplet of year, month and day of
+	 * the month.
```

Comment:
We should also describe what happens when a non-existing calendar date is given.

Reply:
did so

Reply:
please check